styles/variables.scss

```diff
@@ -28,3 +28,6 @@ $odnoklassniki-color: #ed812b;
 $error: #e21017;
 
 $tag-image-height: 48px;
+
+$block-spacing: 60px;
+$paragraph-spacing: 24px;
```

Comment:
How about we put these variables into `styles/pages/article.scss`? Can't see why to make them global.